### ISSUE TITLE
WasapiLoopbackCapture for ASIO ?

### ISSUE THREAD

**Issue by toby11** [naudio/NAudio#995]
Hi

I have been testing WasapiLoopbackCapture  and it works really well - however is there anyway to capture the output of another process when its using ASIO rather thatn Waspi?

I suspect there is not but wanted to check, thanks!

**Comment by KunJakob** [comment]
Were you able to capture audio of a single process using wasapi to begin with? To my knowledge that isn't possible with NAudio currently.

**Comment by markheath** [comment]
Yes, at the moment WasapiLoopbackCapture only captures system audio - process specific capture is a feature I've tried a few times to add but failed very time. Hopefully can get it working sometime.

ASIO does not offer any loopback capture per-se, but some soundcard manufacturers include a loopback feature in their drivers, meaning that the system audio appears as separate inputs you could capture with ASIO. For example the newer Focusrite and Audient interfaces all offer this capability.
